fish_prompt.fish

```diff
@@ -213,6 +213,10 @@ function prompt_status -d "the symbols for a non zero exit status, root and back
     end
 end
 
+function available -a name -d "Check if a function or program is available."
```

Comment:
I've created a new pull request for removing it. 
I do think it might be a good idea to keep the  themes more framework agnostic by making sure your theme works out of the box without OMF.
What do you think?